java/code/src/com/redhat/rhn/taskomatic/task/HubReportDbUpdateWorker.java

```diff
@@ -76,8 +78,19 @@ public void setParentQueue(TaskQueue queue) {
         this.parentQueue = queue;
     }
 
+    private boolean checkRemoteData(Session remoteSession, String tableName) {
+        SelectMode query = generateExistsTable(remoteSession, tableName);
+        DataResult<Map<String, PGobject>> exist = query.execute();
```

Comment:
If possible, I would avoid the explicit reference on `PGobject` and use a more standard `Map<String, Object>`. I'm not very familiar with the internals, but it is also possible that, since you are using a boolean now, it gets converted automatically by the underlying framework.